framework/wazuh/core/manager.py

```diff
@@ -267,7 +267,7 @@ def get_api_conf() -> dict:
     dict
         API configuration.
     """
-    return copy.deepcopy(configuration.api_conf)
+    return copy.deepcopy(configuration.hardcoded_api_config)
```

Comment:
This should return the current API configuration but it's returning default values

Reply:
Fixed in 295ed002d916bd363c2b9284c18e97d79ebe89ed.